pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2770,20 +2783,28 @@ static __always_inline struct pipe_inode_info *get_file_pipe_info(struct file *f
 SEC("raw_tracepoint/sys_enter")
 int tracepoint__raw_syscalls__sys_enter(struct bpf_raw_tracepoint_args *ctx)
 {
-    event_data_t data = {};
-    if (!init_event_data(&data, ctx))
+    struct task_struct *task = (struct task_struct *) bpf_get_current_task();
+    u64 task_id = bpf_get_current_pid_tgid(); // get the tgid only
+    task_info_t *task_info = NULL;
+
+    int zero = 0;
+    config_entry_t *config = (config_entry_t *) bpf_map_lookup_elem(&config_map, &zero);
+    if (config == NULL)
         return 0;
 
-    if (!should_trace(&data))
+    // use config as garbage value here
```

Comment:
This is tricky, because we need the size of the config map value to be greater than the size of the task_info map value. Now it works (task_info map value is ~170 bytes in size, while the config map is ~210 bytes), but we may need to pay attention to this in the future.

Reply:
Yep, I also had the problem with the stack space, why I cotinued with this method.
If you believe config can be a volatile struct for this, maybe we should use another one?